--- ../rose/Makefile.am	2019-06-27 21:22:08.541507461 +0000
+++ Makefile.am	2019-06-27 15:58:50.441546697 +0000
@@ -106,7 +106,7 @@
 install-core: core rosePublicConfig.h
 	$(MAKE) install-data-local
 	$(MAKE) install -C $(top_builddir)/src
-	cp rosePublicConfig.h $(pkgincludedir)
+	cp rosePublicConfig.h $(DESTDIR)$(pkgincludedir)
 
 install-tools: install-core tools
 	$(MAKE) install -C $(top_builddir)/tools
@@ -157,7 +157,7 @@
 # don't want to install anything else (perhaps because it doesn't compile). [Robb Matzke, 2016-03-07]
 install-rose-library: install-data-local install-testprojDATA
 	$(MAKE) -C $(top_builddir)/src install
-	cp rosePublicConfig.h $(pkgincludedir)
+	cp rosePublicConfig.h $(DESTDIR)$(pkgincludedir)
 	$(MAKE) -C $(top_builddir)/python install
 
 # Build a binary release of the ROSE library.  This assumes that you're running in the RMC-2 environment because it uses
